AQ-218 User manual corrected (#93)
MicroscopeIT/aquascope_gui: manual/filtering_pane/README.md

  <img src="static/filtering_pane_tags.png" border=1>
</p>

> To add a tag user must to input text in `Input tag` field and press enter. There can be multiple tags added to the list.

Reviewer: ```suggestion
To add a tag user has to input text in `Input tag` field and press enter. There can be multiple tags added to the list.
```

### Modification Time

This component alows to query the items by their modification time. Its visual behaviour is the same as for [Acquisition Time](#acquisition-time). Its filtering behaviour hovewer is a combination of [Modified by](#modified-by) and [Acquisition Time](#acquisition-time) logics.

#### Filtering behaviour

When `Modification Time` checkbox is enabled it takes part in filtering logic and behaves differently when combined with annotable fields and differently when not.

Additionally in case of querying data by the specific user and specific modification time it is advised to combine this criterion with `Modified by`.

##### Filtering without annotable fields combined

> When `Modification Time` criterion is not combined with annotable fields it matches the items where provided timerange matches modification time of item's __any__  annotable field. Please note that such query is more expensive for the system and it might take a bit longer to be handled.

Reviewer: ```suggestion
When `Modification Time` criterion is not combined with annotable fields it matches the items where provided time range matches modification time of __any__ item's annotable field. Please note that such query is more expensive for the system and it might take a bit longer to be handled.
```

##### Filtering with annotable fields combined

When `Modification Time` criterion is combined with annotable fields it matches the items where provided timerange matches modification times of __all__ combined annotable fields.

### Taxonomy

Taxonomy component allows to query the data by its taxonomic annotation. It consists of 8 combo boxes that correspond (top to bottom) with taxonomic hierarchy: _empire, kingdom, phylum, class, order, family, genus, species_.

<p align="center">
  <img src="static/filtering_pane_taxonomy_plain.png" border=1>
</p>

#### Taxonomy bypass

> For better user experience this component allows user to choose desired value from any level within the chain with no need to specyfing every step on the way. That action is called Taxonomy bypass and can be peformed at any state of the Taxonomy component. Range of possibilities for given levels is adjusted dynamically.

Reviewer: ```suggestion
For better user experience this component allows user to choose desired value from any level within the chain with no need to specify every step on the way. That action is called Taxonomy bypass and can be performed at any state of the Taxonomy component. Range of possibilities for given levels is adjusted dynamically.
```
